draft-ietf-dnsop-svcb-https.md

```diff
@@ -1211,10 +1212,10 @@ with this "https" URL in the "Location" field.  (Receipt of an incompatible Serv
 trigger the redirect behavior.)
 Because HTTPS RRs are received over an often insecure channel (DNS),
 clients MUST NOT place any more trust in this signal than if they
-had received a 307 redirect over cleartext HTTP.
-If this redirection would result in a loss of functionality (e.g. important
-resources that are only available on the "http" origin), the operator MUST
-NOT publish an HTTPS RR.
+had received a 307 redirect over cleartext HTTP.  Where the "http" resource does
+not redirect to the "https" resource or reference the same underlying resource,
+as is commonplace, requests directed to the "http" resource will be processed by
+the "https" resource with potentially unexpected results.
```

Comment:
```suggestion
had received a 307 redirect over cleartext HTTP.  

Publishing an HTTPS RR has the potential to have unexpected results 
or a loss in functionality in cases where the "http" resource neither
redirects to the "https" resource nor reference the same underlying resource.
This is because requests directed to the "http" resource will be processed by
the "https" resource.
```

Rewording to be more clear.  I'm also not sure we should talk about "commonplace" 
since there may also be commonplace cases in the other direction.

I'm not sure if we want to give operator guidance.?

Reply:
I'm not sure the last sentence is a correct description of HTTP redirects.  I would just remove it.

In general, I think this is all extremely obvious, so I don't know how to clarify it.

Reply:
How about this?

```suggestion
had received a 307 redirect over cleartext HTTP.  

Publishing an HTTPS RR has the potential to have unexpected results 
or a loss in functionality in cases where the "http" resource neither
redirects to the "https" resource nor references the same underlying resource.
```

Reply:
That's fine with me.